tosan-httpserver-spring-boot-starter/src/main/java/com/tosan/http/server/starter/statistics/Statistics.java

```diff
@@ -0,0 +1,32 @@
+package com.tosan.http.server.starter.statistics;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author AmirHossein ZamanZade
+ * @since 10/22/2022
+ */
+public class Statistics {
+    private static final ThreadLocal<Statistics> currentStatistics = new ThreadLocal<>();
```

Comment:
static final variable --> CURRENT_STATISTICS, and this thread local doesn't belong to just current statistics...